### DISCUSSION TITLE
How do I get a specific auth? (fortnite:calendar READ)

### DISCUSSION THREAD

**Discussion by Terminxa** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#255] [General]
I'm trying to access the event flags for the Jam Track rotation but I get an error telling me that my login "does not possess the permissions 'fortnite:calendar READ'. I got my access token from fortnitePCGameClient and granted myself a client credential. How can I get the correct authentication to access the event flags?

**Comment by Jaren8r** [comment]
You're correct with using fortnitePCGameClient - however you need to get account credentials as client credentials does not have this perm.
